ui/imports/utils/Style.qml

```diff
@@ -49,6 +49,9 @@ QtObject {
     function emoji(name) {
         return "qrc:/StatusQ/src/assets/twemoji/" + name + ".png";
```

Comment:
Do we need .png emojis? This function is not used anymore, instead of introducing `emojiSvg` maybe would be better to:
```suggestion
        return "qrc:/StatusQ/src/assets/twemoji/" + name + ".svg";
```